package.json

```diff
@@ -49,7 +49,7 @@
     "is-nan": "^1.2.1",
     "memory-cache": "^0.1.6",
     "moment": "^2.14.1",
-    "moment-opening-times": "git://github.com/nhsuk/moment-opening-times.git",
+    "moment-opening-times": "git://github.com/nhsuk/moment-opening-times.git#feature\/performance-improvements",
```

Comment:
I know this is mentioned in the PR notes but thought I'd highlight the fact it needs updating to whatever version is created when the lib changes are merged in so as not to forget to check.